tools/parameters.tfvars

```diff
@@ -304,15 +180,19 @@ extra_conf = {
     MongoDB__TableStorage__PollingDelayMax     = "00:00:10"
     MongoDB__AllowInsecureTls                  = true
     MongoDB__TableStorage__PollingDelay        = "00:00:01"
-    MongoDB__DataRetention                     = "10.00:00:00"
+    MongoDB__DataRetention                     = "1.00:00:00"
```

Comment:
DataRetention change here ? Is it enough for core and Metric exporter ?

Reply:
I just updated based on ArmoniK main's parameters